src/config/index.js

```diff
@@ -1,4 +1,33 @@
+require('dotenv').config();
+
+const { DB_DEV_NAME, DB_USER_NAME, DB_PASSWORD } = process.env;
+
+const dialect = 'postgres';
+
 module.exports = {
```

Comment:
What do you think about using a single connection URL when initializing sequelize, in order to limit the number of  ```env``` variables?

Reply:
I'm not sure how you'd implement it since setting up sequelize requires you to place them seperately,
`new Sequelize('db_name', 'db_username', 'db_password', {
  host: 'localhost',
  dialect: 'postgres',
});`
I'd appreciate if you can send me a resource i can reflect on.
thanks @sneakymaxy

Reply:
Okay, will do that asap

Reply:
https://sequelize.org/master/manual/getting-started

> Okay, will do that asap